onvm/onvm_mgr/main.c

```diff
@@ -253,6 +255,9 @@ tx_thread_main(void *arg) {
 
         RTE_LOG(INFO, APP, "Core %d: TX thread done\n", rte_lcore_id());
 
+        free(tx_mgr->tx_thread_info);
```

Comment:
Should we free(tx_mgr->tx_thread_info->port_tx_bufs) as well?

Reply:
We definitely should, good catch

Reply:
@dennisafa fixed